airbyte-integrations/connectors/source-exchange-rates-tutorial/source_exchange_rates_tutorial/exchange_rates_tutorial.yaml

```diff
@@ -0,0 +1,51 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: [ ]
+  requester:
+    url_base: "https://api.apilayer.com"
+    http_method: "GET"
+    authenticator:
+      type: ApiKeyAuthenticator
+      header: "apikey"
+      api_token: "{{ config['access_key'] }}"
+    request_options_provider:
+      request_parameters:
+        base: "{{ config['base'] }}"
+  stream_slicer:
+    type: "DatetimeStreamSlicer"
+    start_datetime:
+      datetime: "{{ config['start_date'] }}"
+      datetime_format: "%Y-%m-%d"
+    end_datetime:
+      datetime: "{{ now_utc() }}"
+      datetime_format: "%Y-%m-%d %H:%M:%S.%f%z"
+    step: "1d"
+    datetime_format: "%Y-%m-%d"
+    cursor_field: "{{ options['stream_cursor_field'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+    stream_slicer:
+      $ref: "*ref(definitions.stream_slicer)"
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+  rates_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "rates"
+      path: "/exchangerates_data/{{stream_slice['start_time'] or 'latest'}}"
+      stream_cursor_field: "date"
+    primary_key: "timestamp"
```

Comment:
I guess the primary key should be compound, e.g. ["timestamp", "base" ] because you can run multiple syncs against one destination with different bases

Reply:
Should I also change it in `integration_tests/configured_catalog`? And if so what is the correct structure?
`[["timestamp", "base"]]` or
`[["timestamp"], ["base"]]`

Reply:
Yes, they should be the same.

`primary_key = [["timestamp"], ["base"]]`